aiocache/decorators.py

```diff
@@ -338,7 +338,7 @@ async def decorator(
 
     def get_cache_keys(self, f, args, kwargs):
         args_dict = _get_args_dict(f, args, kwargs)
-        keys = args_dict[self.keys_from_attr] or []
+        keys = args_dict.get(self.keys_from_attr, []) or []
```

Comment:
inspect.signature changed and in new versions it parses the defaults of the function correctly